lib/algolia/client.rb

```diff
@@ -552,7 +552,7 @@ def remove_user_id(user_id, request_options = {})
 
     def search_user_id(query, cluster_name = nil, page = nil, hits_per_page = nil, request_options = {})
       body = { :query => query }
-      body[:clusterName] = cluster_name unless cluster_name.nil?
```

Comment:
What is happening today? This argument don't work at all?

Reply:
Indeed, the engine throws an error `{"message":"invalid key 'clusterName'. Expected: query, cluster, appID, apiKey, page, hitsPerPage (near 1:49)","status":400}`